src/Moryx/Serialization/EntryConvert/EntryConvert.cs

```diff
@@ -749,22 +757,100 @@ public static Entry InvokeMethod(object target, MethodEntry methodEntry)
 
         /// <summary>
         /// Invoke a method on the target object using custom customSerialization
+        /// Async methods are executed asynchronous
+        /// </summary>
+        public static Task<Entry> InvokeMethodAsync(object target, MethodEntry methodEntry)
+        {
+            return InvokeMethodAsync(target, methodEntry, Serialization);
+        }
+
+        /// <summary>
+        /// Invoke a method on the target object using custom customSerialization
+        /// Async methods are executed asynchronous
+        /// </summary>
+        public static async Task<Entry> InvokeMethodAsync(object target, MethodEntry methodEntry, ICustomSerialization customSerialization)
+        {
+            var (method, arguments) = GetMethodInfoAndArguments(target, methodEntry, customSerialization);
+            object invokeResult;
+
+            // Also part of MethodEntry but be safer than sorry
+            var isAsync = IsAsyncMethod(method);
+            if (isAsync)
+            {
+                var task = (Task)method.Invoke(target, arguments)!;
+                await task.ConfigureAwait(false);
```

Comment:
If I'm not mistaken, `.ConfigureAwait(false)` is not really relevant in .Net anymore, hence we don't use it anywhere.